commodore/component/compile.py

```diff
@@ -250,6 +251,7 @@ def _prepare_kapitan_inventory(
             "parameters": {
                 "_instance": instance_name,
                 "_base_directory": str(component.target_directory),
+                "_kustomize_wrapper": str(__kustomize_wrapper__),
```

Comment:
Would it be complicated to also add `${_compile_directory}` or `${_output_directory}` or similar?

Reply:
It's possible but will be fragile, since the compile target directory is computed by Kapitan based on the target we're defining here (and in `render_target()` for `catalog compile).

See https://github.com/kapicorp/kapitan/blob/257f53962d375be12da74d9c966e6e423aa68a90/kapitan/targets.py#L420-L468 and https://github.com/kapicorp/kapitan/blob/257f53962d375be12da74d9c966e6e423aa68a90/kapitan/inputs/base.py#L83-L87

Reply:
Actually Kapitan creates a temp directory (cf. https://github.com/kapicorp/kapitan/blob/257f53962d375be12da74d9c966e6e423aa68a90/kapitan/targets.py#L50-L54) which is used as the output directory during compilation, and copies the resulting tree into the final output directory after compilation is completed. We can't really infer the temp directory location ahead of time, so this isn't really feasible in a robust fashion, since the final output directory may not exist at the point when `kustomize` is called.